### ISSUE TITLE
new error with lme4 and weighted model

### ISSUE THREAD

**Issue by ThomasKraft** [strengejacke/ggeffects#131]
Hi, I recently updated your package to the newest version and now old code is giving an error that didn't occur before. The issue occurs when running a multilevel model with lmer() that incorporates weights. When using ggpredict() on such a model, it returns an error saying that confidence intervals could not be computed because:

"Reason: 'Column name 'w' must not be duplicated...."

The problem can be easily reproduced using the following code:
```
y <- rnorm(100, 15, 5)
x <- rnorm(100, 0, 1)
w <- rpois(n = 100, 3)
re1 <- sample(letters[1:10], 100, replace=T)

mod <- lmer(y~x+w + (1|re1), weights=w)
ggpredict(mod, terms=c("x"))
```

Thanks!

**Comment by strengejacke** [comment]
Can you please post your session info? For me, your example works fine:

``` r
library(ggeffects)
library(lme4)
#> Loading required package: Matrix
y <- rnorm(100, 15, 5)
x <- rnorm(100, 0, 1)
w <- rpois(n = 100, 3)
re1 <- sample(letters[1:10], 100, replace=T)

mod <- lmer(y~x+w + (1|re1), weights=w)
ggpredict(mod, terms=c("x"))
#> 
#> # Predicted values of y
#> # x = x
#> 
#>     x | Predicted |   SE |         95% CI
#> -----------------------------------------
#> -3.00 |     16.81 | 1.58 | [13.73, 19.90]
#> -2.50 |     16.70 | 1.40 | [13.95, 19.44]
#> -1.50 |     16.46 | 1.09 | [14.33, 18.59]
#> -0.50 |     16.22 | 0.88 | [14.49, 17.95]
#>  0.00 |     16.10 | 0.85 | [14.44, 17.76]
#>  0.50 |     15.98 | 0.86 | [14.29, 17.67]
#>  1.50 |     15.74 | 1.04 | [13.70, 17.79]
#>  3.00 |     15.38 | 1.51 | [12.42, 18.35]
#> 
#> Adjusted for:
#> *   w = 3.00
#> * re1 = 0 (population-level)
```

<sup>Created on 2020-04-08 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>

**Comment by ThomasKraft** [comment]
Ah OK looks like a local issue then. Here's my SI:

**Comment by strengejacke** [comment]
You may try updating ggeffects from CRAN. Other packages that might be the reason for your issue look up-to-date to me... can you solve that issue? Maybe if you don't load lmerTest? (though it works for me as well when I use lmerTest instead of lme4).

**Comment by ThomasKraft** [comment]
Funny, I had actually loaded lmerTest here only because I thought maybe that had been loaded the last time this was working for me. But that doesn't seem to cause the problem.

The issue is now resolved though, and it looks like the route cause was updating ggeffects from CRAN. Something was going wrong with the package update and I was tricked into thinking it was working when it wasn't. Thank you for the help with this.
